app/api/auth/product-page/page.tsx

```diff
@@ -20,8 +31,46 @@ const AddProduct = () => {
       </div>
       <div className={styles.main}>
         <form onSubmit={handleSubmit}>
-          <label>Add a Product:</label>
-          <input type="text" name="productName" onChange={handleChange} value={productName} style={{color: 'black'}} />
+          <div>
+            <label>Type:</label>
+              <select value={productValue} onChange={handleProductSelect} style={{color: 'black'}}>
+                <option value="">Choose</option>
+                <option value="Option 1">Shoes</option>
+                <option value="Option 2">Long sleeve shirt</option>
+                <option value="Option 3">Short sleeve shirt</option>
+                <option value="Option 4">Pant</option>
+                <option value="Option 5">Skirt</option>
+                <option value="Option 6">Tie</option>
+                <option value="Option 7">Belt</option>
+                <option value="Option 8">Jacket/blazer</option>
+                <option value="Option 9">Handbag</option>
+                <option value="Option 10">Dress</option>
+                <option value="Option 11">Hat</option>
+                <option value="Option 12">Scarf</option>
+                <option value="Option 13">Gown (graduation)</option>
+                <option value="Option 14">Hoodie</option>
+                <option value="Option 15">Sweatshirt</option>
+            </select>
+          </div>
+          <div>
+          <label>Size:</label>
+            <select value={sizeValue} onChange={handleSizeSelect} style={{color: 'black'}}>
```

Comment:
A recommendation but it could be a refactor later as well.
You could use CSS classes or styled-components to handle styling for better separation of concerns